src/state_machine.zig

```diff
@@ -1971,15 +2034,16 @@ pub fn StateMachineType(
                         2 * batch_transfers_limit, // create_transfer(), debit and credit accounts
                     ),
                     .cache_entries_max = options.cache_entries_accounts,
-                    .tree_options_object = .{ .batch_value_count_limit = batch_values_limit.accounts.timestamp },
-                    .tree_options_id = .{ .batch_value_count_limit = batch_values_limit.accounts.id },
-                    .tree_options_index = .{
-                        .user_data_128 = .{ .batch_value_count_limit = batch_values_limit.accounts.user_data_128 },
-                        .user_data_64 = .{ .batch_value_count_limit = batch_values_limit.accounts.user_data_64 },
-                        .user_data_32 = .{ .batch_value_count_limit = batch_values_limit.accounts.user_data_32 },
-                        .ledger = .{ .batch_value_count_limit = batch_values_limit.accounts.ledger },
-                        .code = .{ .batch_value_count_limit = batch_values_limit.accounts.code },
+                    .tree_options_object = .{
+                        .batch_value_count_limit = batch_values_limit.accounts.timestamp,
+                    },
+                    .tree_options_id = .{
+                        .batch_value_count_limit = batch_values_limit.accounts.id,
                     },
+                    .tree_options_index = index_tree_options(
```

Comment:
I am not very happy with this brute-force solution, but I think it's better than what we had here before --- it was very hard to spot a typo in the wall of text we had here before.